### ISSUE TITLE
Extend build terminal in dashboard to full height of viewport

### ISSUE THREAD

**Issue by humphd** [Seneca-CDOT/telescope#2519]
The build terminal (see https://dev.api.telescope.cdot.systems/v1/status/pages/build.html) is only occupying the top portion of the page.  Let's have it extend to fill the available space vertically.

<img width="1254" alt="Screen Shot 2021-11-23 at 6 58 11 PM" src="https://user-images.githubusercontent.com/427398/143147402-718cb809-0d3e-448a-8fd0-6438331dda98.png">

**Comment by Yoda-Canada** [comment]
@humphd I would like to work on this for release 0.4

**Comment by humphd** [comment]
@Yoda-Canada sounds good.  NOTE: this isn't enough for 0.4, so look for some other things to pair it with.

**Comment by Yoda-Canada** [comment]
@humphd where is the code? I find the code in the webpage, but can't find it on GitHub

**Comment by humphd** [comment]
https://github.com/Seneca-CDOT/telescope/blob/master/src/api/status/public/pages/build.html and https://github.com/Seneca-CDOT/telescope/blob/master/src/api/status/public/js/pages/build.js

**Comment by Yoda-Canada** [comment]
I only can find the id="terminal" and I try to "go to definition", but can't find anything. 
![image](https://user-images.githubusercontent.com/55161917/143319573-76551d25-c534-4636-b325-590aecfbbac6.png)

**Comment by humphd** [comment]
Yes, it gets created dynamically by xterm.js at runtime.  You need to stretch the `#terminal` parent element to fill the vertical space, and the terminal should fill it automatically.

**Comment by Yoda-Canada** [comment]
@humphd I try to use the following code to solve this issue, but it doesn't change.  Then I check the container size and the main class size doesn't fit the whole screen. I changed the max-height, but it not useful. Could you give some clues?
```
<div id="terminal">
      <script>
        var term = new Terminal();
        var fitAddon = new FitAddon();
        term.loadAddon(fitAddon);
        term.open(document.getElementById('terminal'));
        fitAddon.fit();
      </script>
</div>
```
![image](https://user-images.githubusercontent.com/55161917/143374410-ca7ff93e-14e6-4988-b440-6ad394535da8.png)

**Comment by humphd** [comment]
Short of solving this myself, I'm not sure.  You need to do some research, work with your classmates to try and dig into things, etc.  You can do this.

**Comment by HyperTHD** [comment]
Might be wrong on this as I'm not the best with css and styling in general but this is what you want right? I also had it align with the dashboard as we

https://user-images.githubusercontent.com/16841702/143526317-44fd647a-54c2-4d83-a5f6-8990c4424e03.mp4

ll

**Comment by Yoda-Canada** [comment]
@HyperTHD I think it is right. Could you tell me how to do it? thx

**Comment by HyperTHD** [comment]
It's the height of the xterm-screen itself. I looked at the new status service related content and couldn't find out where it gets loaded into, but you'd basically want to increase the height of that, and add some border radius so that it ends up rounded at the bottom like the dashboard. Hope this helps @Yoda-Canada

**Comment by Andrewnt219** [comment]
Closed with #2556
